### DISCUSSION TITLE
带图片的memo删除之后，memos_prod.db占用的空间不会自动变小。

### DISCUSSION THREAD

**Discussion by yyysuo** [usememos/memos#67] [General]
1.即使在清空回收站之后，空间也不会恢复。
2.删除带图片memo并清空回收站之后，再重复上传同一张图片，空间继续增长，并没有利用删除文件之后的空间。
3.导出所有memo，重新创建容器并导入memo，空间会恢复。
4.是因为删除memo并没有删除附带的附件吗，那导出的时候其实并没有导出多余的附件。
5.重复导入同一个备份文件，memo会重复，每个memo有唯一id么？能否根据唯一id进行判断？
6.带图片的memo导出后，删除数据库重建容器再导入后，图片会丢失，就是说只export了memo，并没有export图片。

**Comment by boojack** [comment]
图片是一种 resource，可以理解为内置图床，和 memo 之前无任何联系，所以删除 memo 不会删除 resource。如果对这个模块/逻辑有想法，欢迎提 PR~
related issue: https://github.com/usememos/memos/issues/11

**Reply by yyysuo** [reply]
> 图片是一种 resource，可以理解为内置图床，和 memo 之前无任何联系，所以删除 memo 不会删除 resource。如果对这个模块/逻辑有想法，欢迎提 PR~ related issue: #11

小白没能力贡献代码，只能仰望各位大佬了。
